src/components/StatsCard/index.jsx

```diff
@@ -49,7 +49,7 @@ const StatsCard = ({ data, subdata, type }) => (
               ) : (
                 <img
                   src={upArrow}
-                  alt="Up arrow"
+                  alt="Increased to"
```

Comment:
I would suggest using "Increased by" instead since this refers to the increased amount.